### DISCUSSION TITLE
课后作业的代码

### DISCUSSION THREAD

**Discussion by Detailed-x** [zhangyuanes/OperatingSystemExperiments#13] [Q&A]
请问课后作业中2.60和银行家算法的代码什么时候可以放在仓库里

**Comment by wccyzxy** [comment]
同问

**Comment by zhangyuanes** [comment]
> 请问课后作业中2.60和银行家算法的代码什么时候可以放在仓库里

后续老师沟通决定暂不提供代码，给大家带来不便请谅解。仅给出思路：
2.60  读者写者问题
银行家算法 模拟检测流程即可。

**Comment by wccyzxy** [comment]
收到

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

发件人: ***@***.***>
发送时间: 2021年6月30日 15:48
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [zhangyuanes/OperatingSystemExperiments] 课后作业的代码 (#13)


请问课后作业中2.60和银行家算法的代码什么时候可以放在仓库里

后续老师沟通决定暂不提供代码，给大家带来不便请谅解。仅给出思路：
2.60 读者写者问题
银行家算法 模拟检测流程即可。

―
